tools/cli/src/test/java/org/apache/carbondata/tool/CarbonCliTest.java

```diff
@@ -184,6 +184,19 @@ public void testSummaryOutputIndividual() {
     Assert.assertTrue(output.contains(expectedOutput));
   }
 
+  @Test
+  public void testSortColumnsOfSegmentFolder() {
+    String[] args = {"-cmd", "sort_columns", "-p", path};
```

Comment:
Please add test case with multiple index files and also do a alter sort_columns then assert the result